src/transformState.js

```diff
@@ -5,7 +5,23 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    if (actions[i].type === 'addProperties') {
```

Comment:
Nice job. But I've heard that the mentors highly recommend using "switch" instead of "if's". So, I would suggest redoing this part of the task.

Reply:
agree, forgot about switch ))